### PR TITLE
Update "sound" part in contributing guideline

### PR DESCRIPTION
**Pull request by dvkndn**
Eh.. nothing much. I'm just reading the contributing guideline and it looks like you guys forget to complete the sentence at https://github.com/ubuntu/yaru/commit/25136654a68dedd8225808b1dea170cfc073a80b#diff-6a3371457528722a734f3c51d9238c13R7 and I found it on README so I guess I should complete it :D 

Sorry if this is too small of a PR.. I'm trying to go through the issue list to find something to do

### CODE REVIEW

**Comment by madsrh** [comment]
Any contribution is much appreciated @dvkndn. Perhaps this is the first of many 😉 

Thanks!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Yaru consists of one project with 3 distinct parts.
 
 - `gnome-shell` directory is the theme for GNOME Shell. This themes stuff like the calendar widget, the Ubuntu dock, the top panel, the login screen and more. It derives from upstream GNOME Shell theme.
 - `gtk` contains the themes GTK+2 and GTK+3. This specifies how applications like Files, Terminal, Ubuntu Software look. It derives from upstream Adwaita GTK+2 and GTK+3 themes.
-- `sound` contains all sound themes related information. Derives from 
+- `sound` contains all sound themes related information, combines from [WoodenBeaver](https://github.com/madsrh/WoodenBeaver) and [Touch-Remix](https://github.com/madsrh/TouchRemix).
```
Comment (clobrano) [inline CONTRIBUTING.md]: if I get the change right, I would expect something like "combining both..." or "elaborating..."

What do you think?

Comment (dvkndn) [reply CONTRIBUTING.md]: Ah yes, should be "V-ing". So should it be:

> - `sound` contains all sound themes related information, combining both [WoodenBeaver](https://github.com/madsrh/WoodenBeaver) and [Touch-Remix](https://github.com/madsrh/TouchRemix).

Comment (clobrano) [reply CONTRIBUTING.md]: @dvkndn, do you have time to fix this?

Comment (dvkndn) [reply CONTRIBUTING.md]: Sorry I've just updated >< don't know why I missed you guys' reactions and comments

**Comment by clobrano** [comment]
Thanks a lot @dvkndn, merging it now

**Pull request merged**